src/bbs2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -65,6 +68,16 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
         description: "The SSH port (default: 22).",
         getDefaultValue: () => 22);
 
+    public Option<string> SmbUser { get; } = new(
+        name: "--smb-user",
+        description: "The SMB user to be used for downloading the export archive off of the Bitbucket server." +
+                     $"{Environment.NewLine}" +
+                     "Note: The SMB password will also be required in order to connect to a SMB share which has to be set as an environment variable before running the script.");
```

Comment:
I think this is a bit simpler and makes it clear exactly what you need to do.

```suggestion
                     "You must also specify the SMB password using the `SMB_PASSWORD` environment variable.");
```

Reply:
Addressed in d24bdf4f6625c50603738913d1e984396644becc